CMakeLists.txt

```diff
@@ -2,10 +2,22 @@ cmake_minimum_required(VERSION 3.14)
 
 project(zip
   LANGUAGES C
-  VERSION "0.3.0")
+  VERSION "2025.06.18")
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" ${CMAKE_MODULE_PATH})
 set(CMAKE_VERBOSE_MAKEFILE ON)
 
+option(NEW_PROJECT_NAME "Rename project from '${PROJECT_NAME}'" "${PROJECT_NAME}")
+if (NOT (NEW_PROJECT_NAME STREQUAL PROJECT_NAME))
+    set(PROJECT_NAME "${NEW_PROJECT_NAME}")
+endif (NOT (NEW_PROJECT_NAME STREQUAL PROJECT_NAME))
```

Comment:
This not needed for vcpkg.

Reply:
You renamed it though from zip to kubazip

Reply:
"You"? 

*Somebody* did that, years ago. Maybe it did solve a problem, but it didn't do it right IMO. 

What I do is actively rolling this back now in vcpkg. All what vcpkg needs is a different include location (could be obsolete with adopting include dirs) and adjusting the OUTPUT_NAME property. [#46084](https://github.com/microsoft/vcpkg/pull/46084), already merged now.